### ISSUE TITLE
Missing commas on example of config

### ISSUE THREAD

**Issue by ikudosi** [danielme85/laravel-log-to-db#2]
In your example of config logging.channels.database you are missing commas on entries: queue, queue_name, queue_connection.

```
'database' => [
    'driver' => 'custom',
    'via' => danielme85\LaravelLogToDB\LogToDbHandler::class,
    'level' => env('APP_LOG_LEVEL', 'debug'),
    'name' => 'My DB Log',
    'connection' => 'default',
    'collection' => 'log',
    'detailed' => true,
    'queue' => false
    'queue_name' => ''
    'queue_connection' => ''
]
```

**Comment by danielme85** [comment]
Commas in the example logs config array costs extra: for two simple payments of $19.99 you can get your very own commas within 5-8 business days 🛒💳 (thats $19.99 per comma). 

No but seriously: thanks for letting me know. Feel free to do a pull request (its fun even for small things), or I will fix it a little later today.

**Comment by danielme85** [comment]
Fixed with: 062637503d07506be49e476ea2eae99ad7ec2198 Feat: Culture Club.
